### DISCUSSION TITLE
Markdown not showing correctly

### DISCUSSION THREAD

**Discussion by nikero41** [folke/noice.nvim#437] [Q&A]
Markdown in messages does not work for common markdown. only code codes working

![Screenshot 2023-04-20 at 1 45 04 PM](https://user-images.githubusercontent.com/78855607/233343460-9a1c2563-1154-4d1a-bb70-5a30250a5888.png)

I'm I missing something in setup?

I'm using AstroNvim config

**Comment by folke** [comment]
What do you mean? Your screenshot looks exactly how it should look. Also, that's a message from lazy, and lazy sends markdown notifications. Not all notifications will be markdown.
